src/components/timeline/timeline.component.less

```diff
@@ -0,0 +1,227 @@
+                }
+                &.grey4 {
+                    background-color: @timeline-badge-grey4 !important;
+                }
+                &.grey5 {
+                    background-color: @timeline-badge-grey5 !important;
+                }
+            }
+
+            > .timeline-panel {
+                background-color: @timeline-panel-bg;
+                float: left;
+                border: 1px solid @timeline-panel-border;
+                border-radius: 2px;
+                padding: 12px;
+                position: relative;
+                -webkit-box-shadow: @timeline-panel-shadow;
+                box-shadow: @timeline-panel-shadow;
+            }
+
+            > .timeline-panel {
+                &:before {
+                    position: absolute;
+                    top: 40px;
+                    right: -8px;
+                    display: inline-block;
+                    border-top: 8px solid transparent;
+                    border-left: 8px solid @timeline-panel-border;
+                    border-right: 0 solid @timeline-panel-border;
+                    border-bottom: 8px solid transparent;
+                    content: " ";
+                }
+            }
+
+            > .timeline-panel {
+                &:after {
+                    position: absolute;
+                    top: 41px;
+                    right: -14px;
+                    display: inline-block;
+                    border-top: 7px solid transparent;
+                    border-left: 7px solid @timeline-panel-bg;
+                    border-right: 0 solid @timeline-panel-bg;
+                    border-bottom: 7px solid transparent;
+                    content: " ";
+                }
+            }
+
+            > .timeline-badge {
+                color: @timeline-badge-color;
+                width: 100px;
+                max-width: 100px;
+                max-height: 100px;
+                line-height: 25px;
+                font-size: 0.9em;
+                text-align: center;
+                position: absolute;
+                top: 26px;
+                left: 50%;
+                margin-left: -25px;
+                padding: 0 15px 0 15px;
+                background-color: @timeline-bg;
+                z-index: 100;
+                border-radius: 1.25em;
+            }
+
+            &:before,
+            &:after {
+                content: " ";
+                display: table;
+            }
+
+            &:after {
+                clear: both;
+            }
+        }
+    }
+
+    ux-timeline-event {
+        li {
+            .timeline-badge {
+                span {
+                    text-overflow: ellipsis;
+                    white-space: nowrap;
+                    display: block;
+                    overflow: hidden;
+                }
+            }
+        }
+    }
+    
+    @media only screen and (min-width: 768px) {
+        ul.timeline:before {
```

Comment:
Could we apply the same changes to these media queries as these are currently written in CSS style rather than LESS. Eg,

```less
@media only screen and (min-width: 768px) {
    .timeline {
        ux-timeline-event {
             .timeline-badge {
             }
        }
    }
}
```

Reply:
Removed duplication in @media query rules.